content/en/monitors/configuration/_index.md

```diff
@@ -272,15 +272,26 @@ Alerts are grouped automatically based on your selection of the `group by` step
 
 #### Simple alert
 
-`Simple Alert` mode aggregates over all reporting sources. You receive **one alert** when the aggregated value meets the set conditions.
+`Simple Alert` mode triggers a notification by aggregating over all reporting sources. You receive **one alert** when the aggregated value meets the set conditions. For example, you might set up a monitor to notify you if the average CPU usage of all servers exceeds a certain threshold. If that threshold is breached, you'll receive a single notification, regardless of the number of individual servers that contributed to the breach. This can be useful for monitoring broad system trends or behaviors.
+
+
+{{< img src="/monitors/create/simple-alert.png" alt="How monitor notifications are sent in simple alert mode" style="width:100%;">}}
 
 #### Multi alert
 
-`Multi Alert` mode applies the alert to each source according to your group parameters. You receive an alert for **each group** that meets the set conditions. For example, you could group a query looking at a capacity metric by `host` and `device` to receive a separate alert for each host device that is running out of space.
+Unlike `Simple Alert`, a `Multi Alert` monitor triggers individual notifications for each entity in a monitor that breaches the alert threshold.
+
+{{< img src="/monitors/create/multi-alert.png" alt="How monitor notifications are sent in simple alert mode" style="width:100%;">}}
+
+For example, when setting up a monitor to notify you if the P99 latency of all services exceeds a certain threshold, you would receive a **separate** alert for each individual service whose P99 latency exceeded the alert threshold. This can be useful for identifying and addressing specific instances of system or application issues. It allows you to track problems on a more granular level.
+
+When monitoring a large group of entities however, this can create noisy monitors. To mitigate, customize which dimensions trigger alerts to reduce the noise and focus on the alerts that matter most to you. This is called **Notification Grouping**. For instance, you are monitoring the average CPU usage of all your hosts. If you group your query by `service` and `host` but only want alerts to be sent once for each `service` attribute meeting the threshold, remove the `host` attribute from your multi alert options and reduce the number of notifications that are sent.
```

Comment:
That's a tricky one. Notification Grouping (or aggregation) is, in a way, an enhancement of multi alerts, especially when grouping by multiple dimensions. 

FWIW, we don't have to name it at all and just describe it. If you say it's confusing in its current state, then I trust your judgement, since I spent way too much time on this and am likely too close :D 

What do you suggest we do?